### PR TITLE
Move guides to lazy load

### PR DESCRIPTION
**Pull request by trentmwillis**

### CODE REVIEW

**Comment by rwjblue** [comment]
We should probably add some tests here too (using the testing helpers from the asset loader), but I think that may be fine as a follow up...

**Comment by trentmwillis** [comment]
Would like to do that as a follow up (as the whole testing story for lazy Engines is a bit ill-defined at this point in time). The existing Acceptance tests should keep us above water in terms of making sure we don't break the site.

File: app/transitions.js [hunk 1]
```diff
@@ -7,20 +7,8 @@ export default function transitionMap() {
 
   this.transition(
     this.fromRoute('index'),
-    this.toRoute('guide'),
+    this.toRoute('application'),
```
Comment (sudowork) [inline app/transitions.js:10]: @trentmwillis How would you end up handling multiple engines? Would `'application'` be a conflicting route name?

Comment (trentmwillis) [reply app/transitions.js:10]: Yep. Mentioned this during my talk last night, this actually needs to be fixed in liquid-fire. I'll go open an issue for it now.

Comment (trentmwillis) [reply app/transitions.js:10]: Opened: https://github.com/ember-animation/liquid-fire/issues/526

**Pull request merged**